src/main/scala/akkasmpp/demos/CliSmpp.scala

```diff
@@ -33,32 +33,47 @@ object CliSmpp extends App {
     """.stripMargin
 
   def receiveCallback: SmppClient.ClientReceive = {
+    // ack messages so they don't repeat
+    case sms: DeliverSm => DeliverSmResp(CommandStatus.ESME_ROK, sms.sequenceNumber, None )
     case p: Pdu => GenericNack(CommandStatus.ESME_RINVSYSTYP, p.sequenceNumber)
   }
+
   val logger = new PduLogger {
     override def logIncoming(pdu: Pdu): Unit = println("Incoming: " + pdu)
     override def logOutgoing(pdu: Pdu): Unit = println("Sending : " + pdu)
   }
 
-  def getDest() = COctetString.ascii("+15094302095")
+  def getDest(num: String= "+15094302095") = COctetString.ascii(num)
 
-  def getSrc() = COctetString.ascii("+12512161914")
 
-  def getShortMsg() = OctetString.fromBytes(DefaultGsmCharset.encode("This was a triumph."))
+  def getShortMsg(msg: String = "This was a triumph.") = OctetString.fromBytes(DefaultGsmCharset.encode(msg))
 
   val pduBuilder = new PduBuilder()
 
-  case class RunConfig(server: String, port: Int, ssl: Boolean) {
+  case class RunConfig(
+      server: String = "localhost",
+      port: Int = 2775,
+      ssl: Boolean= false, 
+      systemId: String = "someasfd",
+      password: String = "asfd",
+      shortCode: COctetString = COctetString.ascii("+12512161914")) {
```

Comment:
this is actually a longcode, not a shortcode. The name of the field should probably be like `souceAddr` or `from`